frontend2/src/views/Submissions.tsx

```diff
@@ -23,6 +31,21 @@ const Submissions: React.FC = () => {
   const [tourneySubData, setTourneySubData] =
     useState<Maybe<TournamentSubmission[]>>();
 
+  const [submitting, setSubmitting] = useState<boolean>(false);
+  const [subError, setSubError] = useState<Maybe<string>>();
+
+  const onSubmit: SubmitHandler<SubmissionUploadRequest> = async (data) => {
```

Comment:
I never hit this function 🥲

Reply:
see my comment below about changing how handleSubmit is called